format_strawberryfield.install

```diff
@@ -213,4 +213,111 @@ function format_strawberryfield_update_8702() {
   }
   $message = "All Metadata Display Entities in Configurations updated to use UUIDs";
   return $message;
-}
\ No newline at end of file
+}
+
+/**
+ * Implements hook_update_dependencies().
+ */
+function format_strawberryfield_update_dependencies() {
+  // The update function that adds the status field must run after
+  // content_translation_update_8400() which fixes NULL values for the
+  // 'content_translation_status' field.
+  if (\Drupal::moduleHandler()->moduleExists('content_translation')) {
+    $dependencies['format_strawberryfield'][8400] = [
+      'content_translation' => 8400,
+    ];
+
+    return $dependencies;
+  }
+}
+
+/**
+ * Add the 'published' and revisionable metadata fields to my_entity.
+ */
+function my_module_update_8400() {
```

Comment:
This function needs to be named format_strawberryfield_ (replace the my_module with this module's name) and the numbering needs to be something after the previous one which was 

`function format_strawberryfield_update_8702() `

So format_strawberryfield_update_8703?